flow/project.py

```diff
@@ -2519,6 +2605,12 @@ class MyProject(FlowProject):
             'status',
             parents=[base_parser])
         self._add_print_status_args(parser_status)
+        parser_status.add_argument(
```

Comment:
Adding a way to specify that we should not exclude `signac-flow`'s own code would be helpful for developers and users who may want to avoid certain functions that could be more costly for whatever reason. This would also help us find pressure points in the code.